bridge/server/app.js

```diff
@@ -27,6 +33,59 @@ if(!integrationsPageLink) {
   integrationsPageLink = "https://get.keptn.sh/integrations.html";
 }
 
+
+try {
+  console.log("Installing default Look-and-Feel");
+
+  const destDir = path.join(__dirname, '../dist/assets/branding');
+  const srcDir = path.join(__dirname, '../client/assets/default-branding');
+  const brandingFiles = ["app-config.json", "logo.png", "logo_inverted.png"];
+
+  if(!fs.existsSync(destDir)) {
+    fs.mkdirSync(destDir, { recursive: true });
+  }
+
+  brandingFiles.forEach((file) => {
+    fs.copyFileSync(path.join(srcDir, file), path.join(destDir, file));
+  });
+} catch (e) {
+  console.error(`Error while downloading custom Look-and-Feel file. Cause : ${e}`);
+  process.exit(1);
+}
+if(lookAndFeelUrl) {
+  try {
+    console.log("Downloading custom Look-and-Feel file from", lookAndFeelUrl);
+
+    const destDir = path.join(__dirname, '../dist/assets/branding');
+    const destFile = path.join(destDir, '/lookandfeel.zip');
+
+    if(!fs.existsSync(destDir)) {
+      fs.mkdirSync(destDir, { recursive: true });
+    }
+    let file = fs.createWriteStream(destFile);
+    let parsedUrl = urlParser.parse(lookAndFeelUrl);
+    let lib = parsedUrl.protocol === "https:" ? https : http;
+
+    lib.get(lookAndFeelUrl, function(response) {
+      response.pipe(file);
+      file.on('finish', function() {
+        file.close(() => {
+          try {
+            let zip = new admZip(destFile);
+            zip.extractAllTo(destDir, true);
+          } catch (err) {
+            console.error(`[ERROR] Error while extracting custom Look-and-Feel file. ${err}`);
+          }
+        });
+      });
+    }).on('error', function(err) {
+      fs.unlink(destFile);
```

Comment:
The server crashes if an invalid url is provided
```
TypeError [ERR_INVALID_CALLBACK]: Callback must be a function. Received undefined
```

Reply:
This should work now as well. If the file is not downloadable, extractable or the provided URL is invalid at all, there will be an error log message, but the Bridge should start and work with the default look and feel.